scalafmt-tests/src/test/resources/align/DefaultWithAlign.stat

```diff
@@ -752,3 +752,81 @@ object a {
   private[this] val redirectClass =
     pbClass.getClasses find (_.getSimpleName == "Redirect")
 }
+<<< multiline with blanks
+align.preset = most
+===
+object a {
```

Comment:
For some reason if you add a comment, the problem is coming back. For example:
```scala
  val foo: Int = {
    val x = ???
    42
  }

  val bar: Int = {
    for { x <- "hey" } yield x
    42
  }

  val fooBig: Int    = {
    42
  }

  // a
  val fooBigger: Int = {
    42
  }
```
I added the `// a` and it tried to align with the code above.

Reply:
ok, will take a look. comment probably determines the alignment container incorrectly.